react/src/components/atoms/forms/InputTextFuzzy/index.js

```diff
@@ -261,7 +267,9 @@ InputTextFuzzy.propTypes = {
   /** The id of the the input tag */
   inputId: PropTypes.string,
   /** By default all options will be rendered as suggestions on input focus */
-  renderDefaultSuggestion: PropTypes.bool
+  renderDefaultSuggestion: PropTypes.bool,
+  /** When true, calls preventDefault on the onKeyDown SyntheticEvent for the input element. */
+  onKeyDownPreventDefault: PropTypes.bool
```

Comment:
```suggestion
  onEmptyEnterPreventDefault: PropTypes.bool
```
can we change the name of this prop to be more descriptive of what it's doing?